src/frontend/pages/chat/sidebar/conversation/conversation-list.js

```diff
@@ -413,6 +421,9 @@ class ConversationList extends LitElement {
       this.newConversationList
     );
 
+    if (this.newConversationListFiltered.length === 0)
```

Comment:
![image](https://github.com/Studiofarma/Infolab/assets/104075123/b59c0e10-d53d-477c-b8c9-4c523df707c0)
La scritta "nessun risultato" deve apparire solamente quando si è in focus con l'input di ricerca.